sigs/sig-serverless.md

```diff
@@ -0,0 +1,139 @@
+   to be within the scope of this SIG (see below)
+4. Identify and fill gaps in the landscape of CNCF Projects within scope
+   either through encouragement of existing projects to join the CNCF
+   or through development of new artifacts (e.g. specs, code, papers).
+5. Maintain any "living" documentation, such as landscape and state of the
+   community documents.
+
+Specific SIG deliverables are as per the above, and the [general SIG
+responsibilities set out by the CNCF
+ TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+## Current CNCF Projects Considered To Be Within The Scope Of This SIG
+
+* CloudEvents
+* Serverless Workflow
+* CloudEvents Discovery API
+* CloudEvents Subscription API
+
+## Interfaces With Other Related Groups
+
+It is not always clear where the line is between
+Serverless and other cloud native hosting technologies, therefore the exact
+choice of a project being within scope of SIG-Serverless versus one of the
+other CNCF SIGs will need to be consider on a case by case basis. Often the
+decision might be based on the project's relationship to existing projects
+within the SIG.
+
+*   **[Several Kubernetes SIGs](https://github.com/kubernetes/community)**
+    cover Kubernetes-specific workload, scheduling, autoscaling, execution
+    and other related abstractions, interfaces, and implementations of
+    these interfaces.  We will maintain communication with these Kubernetes
+    SIGs where needed.  Our aim is to avoid unnecessary duplication of
+    effort by the two groups and maintain clear and consistent messaging
+    to our end user community and projects.
+*   **[CNCF Security SIG](https://github.com/cncf/sig-security)**
+    works on the more general area of cloud-native security including
+    authentication, authorization, encryption, accounting, auditing, and
+    related topics.  We defer as much as possible to this group to deal
+    with general security-related issues and liaise closely with them on
+    how to deal with security areas where these arise.
+*   **[CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery)**
+    is focussed on the development, deployment, operation and testing of
+    cloud-native applications. We collaborate with this SIG where it
+    pertains to helping to ensure that the required underlying workload
+    execution abstractions and mechanisms are suitably provided to support
+    these application-level delivery needs.
+
+## Operating Model
+
+This SIG follows the
+[standard operating
+ guidelines](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#operating-model)
+provided by the TOC unless otherwise stated here.
+
+**Current TOC Liaison:**  ????
+
+**Co-Chairs:** [Doug Davis](https://github.com/duglin/),
+  [Mark Peek](https://github.com/markpeek/),
+  [Ken Owens](https://github.com/kenowens12)
+
+**Tech Leads:** [Doug Davis](https://github.com/duglin/),
```

Comment:
hmm, it's not clear to me if the definition of "team leads" in that doc are referring to "SIG tech leads" or "project tech leads".  I don't think project leads should be listed in the SIG charter since the charter/README of each project should do that.  And, so I'm not sure what a SIG tech lead does that wouldn't be covered by a SIG chair. If the idea is that there's some kind of tech liaison between the SIG and the project, then I'd prefer to just have the project leads take on that role.

The names picked were based on how things work today in the WG.

Reply:
They are SIG Tech Leads